### PR TITLE
f-status-banner@v0.1.0 - Base component.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
Base component to replace our static 404 page or any other status page.

### Added
- Base component generator files.

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [ ] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: packages/components/organisms/f-status-banner/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<div align="center">
+
+# f-status-banner</h1>
```
Comment (DamianMullins) [inline packages/components/organisms/f-status-banner/README.md]: Not sure what's happened here but I think you need to use the `h1`?

File: packages/components/organisms/f-status-banner/src/components/StatusBanner.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        data-test-id="statusBanner">
+        {{ copy.text }}
+    </div>
+</template>
+
+<script>
+import { globalisationServices } from '@justeat/f-services';
+import tenantConfigs from '../tenants';
+/* eslint-disable */
+// import StatusBannerServiceApi from '../services/StatusBannerServiceApi';
```
Comment (DamianMullins) [inline packages/components/organisms/f-status-banner/src/components/StatusBanner.vue:13]: Do you need this?

File: packages/components/organisms/f-status-banner/src/services/StatusBannerServiceApi.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+export default {
+    async dummyService (url, tenant, data) {
```
Comment (DamianMullins) [inline packages/components/organisms/f-status-banner/src/services/StatusBannerServiceApi.js:4]: Is this a placeholder for future code?

File: packages/components/organisms/f-status-banner/README.md (hunk 1 above)
Comment (ashleynolan) [reply packages/components/organisms/f-status-banner/README.md]: Can just remove the closing tag – I think this was a refactoring that was made. I'll update the generator too

File: packages/components/organisms/f-status-banner/README.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+```sh
+$ cd packages/components//StatusBanner
+```
+
+## Testing
+
+### Unit, Integration and Contract
+
+To test all components, run from root directory.
+To test only `f-status-banner`, run from the `./fozzie-components/packages/f-status-banner` directory.
```
Comment (ashleynolan) [inline packages/components/organisms/f-status-banner/README.md]:
```suggestion
To test only `f-status-banner`, run from the `./fozzie-components/packages/components/organisms/f-status-banner` directory.
```

File: packages/components/organisms/f-status-banner/src/components/StatusBanner.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-status-banner/src/components/StatusBanner.vue:13]: I'll probably end up removing this in the next pr tbh. Just left it in incase I end up needing it for the next set of changes.

File: packages/components/organisms/f-status-banner/src/services/StatusBannerServiceApi.js (hunk 3 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-status-banner/src/services/StatusBannerServiceApi.js:4]: Similar to above. Left in incase I need it. Will delete in the next PR if not.

File: packages/components/organisms/f-status-banner/README.md (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-status-banner/README.md]: Fixed.

File: packages/components/organisms/f-status-banner/README.md (hunk 4 above)
Comment (ashleynolan) [reply packages/components/organisms/f-status-banner/README.md]: I'll update this path in the generator too

File: packages/components/organisms/f-status-banner/README.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+yarn storybook:serve-static
+yarn test-component:chrome
+```
+
+OR
+
+```bash
+# Run Component tests for f-status-banner
+# Note: Ensure Storybook is not running when running the following commands
+cd ./fozzie-components/packages/f-status-banner
```
Comment (ashleynolan) [inline packages/components/organisms/f-status-banner/README.md]:
```suggestion
cd ./fozzie-components/packages/components/organisms/f-status-banner
```

**Review by ashleynolan — DISMISSED** [review]

File: packages/components/organisms/f-status-banner/README.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+```sh
+$ git clone git@github.com:justeat/fozzie-components.git
+$ cd fozzie-components
+$ yarn
+```
+
+Change directory to the `StatusBanner` package:
+
+```sh
+$ cd packages/components//StatusBanner
```
Comment (zlatinivanov-je) [inline packages/components/organisms/f-status-banner/README.md]: Path seems incorrect

**Review by zlatinivanov-je — DISMISSED** [review]
Just one little comment in the README file

File: packages/components/organisms/f-status-banner/README.md (hunk 6 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-status-banner/README.md]: Yup i think the line above is wrong too. Will fix it.

**Review by DamianMullins — DISMISSED** [review]

**Review by ashleynolan — DISMISSED** [review]

**Review by ashleynolan — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**